api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -110,8 +110,19 @@ message ProcessingRequest {
     HttpTrailers response_trailers = 7;
   }
 
+  // [#not-implemented-hide:]
+  // TODO(jbohanon) reserve field as part of rework detailed in https://github.com/envoyproxy/envoy/issues/32125
```

Comment:
@htuch , per discussion in the issue thread, shall we just keep this field as it is, maintaining a clear separation between attributes derived from the connection/request/resp vs those pushed through config or internal state?